packages/contracts/package.json

```diff
@@ -29,6 +29,7 @@
     "test": "yarn run test:contracts",
     "test:contracts": "hardhat test --show-stack-traces",
     "test:gas": "hardhat test \"test/contracts/OVM/execution/OVM_StateManager.gas-spec.ts\" --no-compile --show-stack-traces",
+    "test:coverage": "NODE_OPTIONS=--max_old_space_size=8192 hardhat coverage",
```

Comment:
Solidity coverage requires a lot of extra memory.

Reply:
Can we do coverage threshold checking with `istanbul` perhaps? It will ensure at least we never drop below the current level and only grow the code covered

Reply:
I think we could setup a .codecov.yml file to handle that, based on these docs:

https://docs.codecov.io/docs/commit-status#basic-configuration

And agree that'd be good to do.

Reply:
But this wasn't included here :(